.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "develop" ]
+  schedule:
+    - cron: '17 19 * * 5'
```

Is it on purpose to run this pipeline "At 19:17 on Friday" ? :)
I guess it is not, so let's have a more generic timing such as `0 0 * * 1` and comment what it means:
```suggestion
    # https://crontab.guru/#0_0_*_*_1
    - cron: '0 0 * * 1'
```